.github/workflows/tests.yaml

```diff
@@ -21,14 +21,15 @@ on:
     workflow_dispatch:
 
 concurrency:
-    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name ==
+    group:
```

Comment:
could we separate out the whitespace changes here? this adds a bit of noise to the PR making it harder to see "real" changes

Reply:
Updated.

For some reason I can turn off format on save in VS Code.